src/tooling/piral-cli/src/common/config.ts

```diff
@@ -45,15 +45,15 @@ export interface PiralCliConfig {
   /**
    * Automatically open the browser.
    */
-  open?: boolean;
+  openBrowser?: boolean;
   /**
    * Port number.
    */
   port?: number;
   /**
    * Template language.
    */
-  language?: SourceLanguage.ts;
+  sourceLanguage?: SourceLanguage.ts;
```

Comment:
It's still `SourceLanguage.ts` but I think it should be `SourceLanguage`, right? I mean the type should be either 'js' or 'ts' not only 'ts'.

Reply:
Fixed.